.github/pull_request_template.md

```diff
@@ -2,20 +2,23 @@
 
 ## _Soft_ Schema Freeze
 
-The `nmdc-schema` and `berkeley-schema-fy24` schemas are under a soft freeze, which means changes **should not** be made that have any downstream implications. To ensure this, all PRs created creating during the freeze will be closely reviewed with **every** component of the NMDC system in mind.
+The `nmdc-schema` and `berkeley-schema-fy24` schemas are under a soft freeze, which means changes **should not** be made that have any downstream implications. To ensure this, all PRs created during the freeze will be closely reviewed with **every** component of the NMDC system in mind.
 
 ## Reviewers
 
-To ensure no changes are made unexpectedly, PR creators will request reviews from _all_ [Berkeley Schema Roll Out task coordinators](https://docs.google.com/document/d/1XXN1YuaBuSkxPXeiLKm5YxYzXTamBPQrzzeLhlh7PWs/edit#heading=h.u52g8v319adh).
+To ensure no changes are made unexpectedly, PR creators will us this PR template to tag and notify all task coordinatos. Review should be specificlly requested from _all_ [Berkeley Schema Roll Out task coordinators](https://docs.google.com/document/d/1XXN1YuaBuSkxPXeiLKm5YxYzXTamBPQrzzeLhlh7PWs/edit#heading=h.u52g8v319adh) that you know to be affected by this PR.
 
 We expect task coordinators to review PRs and provide feedback/approval within 1 week of when they are identified as reviewers. 
 
-PRs will **NOT** be merged until all task coordinators (or one of their delegates) have approved. 
+PRs will **NOT** be merged until all task coordinators (or one of their delegates) have approved either officially as a review, in a metting, or via a comment. 
 
 Expedition, questions, and discussion can happen at any meeting.
 
 Delays in review & merging should be addressed in meetings or with NMDC leadership.
 
+@mslarae13, @eecavanna,  @shreddd, @naglepuff, @aclum, @mbthornton-lbl, @corilo, @kheal, @SamuelPurvine, @sujaypatil96, @brynnz22 
```

Comment:
I propose including the names of the "subsystems" that each person is the coordinator for; as a reminder to the person filling out the PR template.

Reply:
For example:
- @eecavanna - Runtime and database migrations
- @shreddd - MongoDB database*
- ...

\* Like in the Gantt chart, I am OK with "MongoDB database" being listed as one of my areas of ownership (in which case, I would engage @shreddd as needed).

Reply:
I think I got the changes in. Anything else to add?

Reply:
Thanks! I'll post a final suggestion in 15 minutes, which will change the formatting and also rephrase the line about me and @shreddd to emphasize that I will pull him in as needed; not by default).